onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -572,20 +595,18 @@ void OpTester::Run(ExpectResult expect_result,
               }
             }
 
-            if (!valid)
-              break;
+            if (!valid) break;
           }
         }
 
-        if (!valid)
-          continue;
+        if (!valid) continue;
 
```

Comment:
FWIW I was debugging something today and set a breakpoint on the line that previously only had the 'continue' statement

Reply:
Fixed.